app/controllers/CommendSectionController.scala

```diff
@@ -0,0 +1,55 @@
+package controllers
+
+import akka.http.scaladsl.model.headers.LinkParams.title
+import javax.inject.Inject
+import models.Commends
+import play.api.mvc._
+import play.modules.reactivemongo.{MongoController, ReactiveMongoApi, ReactiveMongoComponents}
+import services.ReleasedServices
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use brackets to group imports from the same package](https://app.codacy.com/manual/RafSobolQAC/cinema-team-one/pullRequest?prid=5276325)